### ISSUE TITLE
JavasScript error opening new window (macOS Mojave)

### ISSUE THREAD

**Issue by gdamore** [zeit/hyper#3299]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version (2.0.0)
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS Mojave
- **Hyper.app version**: 2.0.0
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: (started with it removed, it is the default)
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla
## Issue

Just tried to install stock Hyper first time user on macOS Mojave.  Trying to open a shell window, I get a JavaScript error, and the terminal window closes:

```
Uncaught Exception:
TypeError: Cannot read property 'resize' of undefined
    at Server.Window.rpc.on (/private/var/folders/48/hq3502h10mg0_1t71j3j25wm0000gn/T/AppTranslocation/FE61E0F7-C5EE-442F-B882-960E3A3D5D11/d/Hyper.app/Contents/Resources/app.asar/ui/window.js:146:14)
    at emitOne (events.js:115:13)
    at Server.emit (events.js:210:7)
    at Server.ipcListener (/private/var/folders/48/hq3502h10mg0_1t71j3j25wm0000gn/T/AppTranslocation/FE61E0F7-C5EE-442F-B882-960E3A3D5D11/d/Hyper.app/Contents/Resources/app.asar/rpc.js:33:11)
    at emitTwo (events.js:125:13)
    at EventEmitter.emit (events.js:213:7)
    at WebContents.<anonymous> (/private/var/folders/48/hq3502h10mg0_1t71j3j25wm0000gn/T/AppTranslocation/FE61E0F7-C5EE-442F-B882-960E3A3D5D11/d/Hyper.app/Contents/Resources/electron.asar/browser/api/web-contents.js:266:13)
    at emitTwo (events.js:125:13)
    at WebContents.emit (events.js:213:7)
```

Basically, it seems impossible for me to even try this application.

**Comment by 1Reinier** [comment]
Same issue here

**Comment by robertgrzonka** [comment]
How did you install it? Did you `brew` it or download from [webpage](https://hyper.is/)?

**Comment by 1Reinier** [comment]
Upon installing Mojave, some brew links got corrupted. I noticed because none of my terminals were working (all default to `zsh`). I solved it as such: 
Using the bash shell, I reinstalled and relinked `gdbm`, `pcre` and `zsh` with brew.

**Comment by L-A** [comment]
I suddenly have the same thing. I've been using Hyper daily for a few years, including the last months on Mojave.

I ran `brew doctor` to check on @1Reinier's suggestion. I don't have the same issue with links or my shell. OS X's Terminal app and iTerm both work as expected.

I get exactly the same error as @gdamore. Hyper boots but can't open a window.

**Comment by timothyjoelwright** [comment]
I'm also having this issue recently on macOS High Sierra 10.13.6

**Comment by badmike** [comment]
Same issue started today, worked fine the last weeks with daily usage 

Hyper v2.0.0 on windows 10 (1803 build 17134.407)

moving the main window while the error message showed up, fixed it for me somehow.

**Comment by jefheaton** [comment]
tldr; The default shell was fish, I did not have fish installed, so I was able to fix it by installing the `fish` shell

- I poked around and saw other issues with this same error calling for the user to uninstall broken plugins
- I was still able to open the app, it just wouldn't show a shell for more than a flash, and occasionally would show the above error
- Since I could open the app, I went to "Hyper" and then "Preferences" which opened the `.hyper.js`, (located in your home folder)
- There I noticed line 100: `shell: '/usr/local/bin/fish',`, which I confirmed I did not have
- I used homebrew to install fish `brew install fish`, and copied `/usr/local/bin/fish` into my `/etc/shells` file

**Comment by L-A** [comment]
I had commented _all_ lines in `.hyper.js` in an attempt to solve this, and was still getting the symptoms described. I believe the issue is linked to these two circumstances together:

- Having changed the default shell
- Using a `.hyper.js` created before `2.0.0`

Commenting everything in `.hyper.js` did _not_ change the situation. However, deleting the file completely and making Hyper generate a new one cleared the issue.
